frontend/WikiContrib-Frontend/src/query.js

```diff
@@ -533,7 +534,10 @@ export class Query extends Component {
                       animation="fade"
                     >
                       <React.Fragment>
-                        <h1><Header className="title">WikiContrib</Header></h1>
+                        <Header className="title_and_logo">
+                          <img className="logo" src={logo} alt="wikicontrib logo"/>
+                          <h1 className="title">WikiContrib</h1>
+                        </Header>
                         <h2 className="accounts">
                           Get the Contributions of your fellow Wikimedians,
```

Comment:
Change the entire description to "Visualize your technical contributions or a fellow Wikimedian's to Wikimedia projects!"

Reply:
Also, unrelated but `i` icon on hover in the navbar doesn't say anything about Github.